R/pairwise-comparisons.R

```diff
@@ -132,16 +154,24 @@ pairwise_comparison <- function(
   # scores will simply be 1.
   if (setequal(by, forecast_unit)) {
     if (setequal(by, "model")) {
-      warning(
-        "`by` is set to 'model', which is also the unit of a single forecast. ",
-        "This doesn't look right. All relative skill scores will be equal to 1."
+      #nolint start: keyword_quote_linter
+      cli_warn(
+        c(
+          "!" = "`by` is set to 'model', which is also the unit of a single
+         forecast. This doesn't look right.",
+        "i" = "All relative skill scores will be equal to 1."
+        )
       )
     } else {
       by <- "model"
-      message(
-        "relative skill can only be computed if `by` is different from the ",
-        "unit of a single forecast. `by` was set to 'model'"
+      cli_inform(
+        c(
+          "!" = "relative skill can only be computed if `by` is different from the
+        unit of a single forecast.",
```

Comment:
I still don't really understand how linebreaks work in `cli`. Should this be a linebreak? Is it even easily possible to wrap code in the 80char line limit?

Reply:
As far as I know, it concatenates the string and removes all `\n` values, so basically, it prints a single string wrapped in the console. To get new lines, you need to use bullets, paragraphs, or texts (`cli_text`).

> Should this be a linebreak? Is it even easily possible to wrap code in the 80char line limit?

The line break here is just visual and to escape the 80-character limit but it is printed as a single string.

Reply:
Yes, exactly.